nexuse2e-webapp/src/main/webapp/html/nexuse2e.css

```diff
@@ -501,4 +501,168 @@ td.BUTTON_RIGHT {
 
 .NexusMessage {
 	background-color: #CCFFCC;
+}
+
+.statistics {
```

Comment:
The last heading was "OLD NEXUS styles". So I think new we should add a new Headline or move this above the other headline.

Reply:
Moved those rules above "OLD NEXUS styles".